### PR TITLE
[excel] (PivotTable) Update PivotTable article with better object model description

### PR DESCRIPTION
**Pull request by AlexJerabek**
I took another pass at this article to better articulate the relationship between PivotTable components (hierarchies, fields, items, and layouts).

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9728cc3](https://github.com/OfficeDev/office-js-docs-pr/commits/9728cc3eee90f58bc5d9925244e90747f51e94b9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-pivottables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Pivot/docs/excel/excel-add-ins-pivottables.md) | :white_check_mark:Succeeded | |
[docs/images/excel-pivots-raw-data.png](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Pivot/docs/images/excel-pivots-raw-data.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001222223510301-1553%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SkwZ%2Fhl7ZY0jUs34L%2Br6N7jiRBm27OY44H0ZiB3Pqwg%3D&st=2020-01-22T22%3A20%3A22Z&se=2020-02-22T22%3A25%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-pivottables.md [hunk 1]
```diff
@@ -1,37 +1,49 @@
...
 ![A collection of fruit sales of different types from different farms.](../images/excel-pivots-raw-data.png)
 
-This data has five hierarchies: **Farms**, **Type**, **Classification**, **Crates Sold at Farm**, and **Crates Sold Wholesale**. Each hierarchy can only exist in one of the four categories. If **Type** is added to column hierarchies and then added to row hierarchies, it only remains in the latter.
+This fruit farm sales data will be used to make a PivotTable. Each column, such as **Types**, is a `PivotHierarchy`. The **Types** hierarchy contains the **Types** field. The **Types** field contains the items **Apple**, **Kiwi**, **Lemon**, **Lime**, and **Orange**.
+
+### Hierarchies
+
+PivotTables are organized based on four hierarchy categories: row, column, data, and filter.
+
+The farm data shown earlier has five hierarchies: **Farms**, **Type**, **Classification**, **Crates Sold at Farm**, and **Crates Sold Wholesale**. Each hierarchy can only exist in one of the four categories. If **Type** is added to column hierarchies and then added to row hierarchies, it only remains in the latter.
```
Comment (Rick-Kirkham) [inline docs/excel/excel-add-ins-pivottables.md]: It isn't clear to me what "the four categories" refers to. I gather from the next sentence that one is "row" and another is "column". What are the other two?

Comment (AlexJerabek) [reply docs/excel/excel-add-ins-pivottables.md]: Thanks @Rick-Kirkham. The four hierarchies are listed in the preceding sentence: "PivotTables are organized based on four hierarchy categories: row, column, data, and filter." I'll edit that to make it more obvious.

Comment (Rick-Kirkham) [reply docs/excel/excel-add-ins-pivottables.md]: Sorry. My bad.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[15406b1](https://github.com/OfficeDev/office-js-docs-pr/commits/15406b1a2949f3298139ccbaf70a2f858690b3ed)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-pivottables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Pivot/docs/excel/excel-add-ins-pivottables.md) | :white_check_mark:Succeeded | |
[docs/images/excel-pivots-raw-data.png](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Pivot/docs/images/excel-pivots-raw-data.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001242256130393-1553%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=sMIKaEXfJwqpPqes3prYcbytMKu14MXja0Bpt%2BtjTPs%3D&st=2020-01-24T22%3A52%3A24Z&se=2020-02-24T22%3A57%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**